webserver/static/gulpfile.js

```diff
@@ -42,6 +42,9 @@ function writeResource(stream) {
     }))
     .on('finish', function () {
       deferred.resolve();
+    })
+    .on('error', function (error) {
```

Comment:
I think that needs to be directly after `stream`, because it only catches errors that occur immediately before it. Unless you tested it and it works, in which case ignore me. :)